src/Cnab.Bundle/Bundle.cs

```diff
@@ -9,6 +9,12 @@ namespace Cnab.Bundle
 {
     public class Bundle
     {
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("version")]
+        public string Version { get; set; }
```

Comment:
These two properties were missing in the initial bootstrapping of the Bundle type.